### PR TITLE
Dash daq docs

### PR DESCRIPTION
**Pull request by shammamah**
In accordance with https://github.com/plotly/streambed/issues/11974, adding Dash DAQ to Dash Docs.

### CODE REVIEW

**Comment by xhlulu** [comment]
The PR is returning an errorhttps://dash-docs-pr-350.herokuapp.com/

File: tutorial/chapter_index.py [hunk 1]
```diff
@@ -154,6 +156,15 @@
                        'and more.'
     },
 
+    'dashdaq': {
+        'url': '/dash-daq',
+        'content': daq.layout,
+        'name': 'Dash DAQ Components',
+        'description': 'Beautifully styled, premium components for data \
```
Comment (xhlulu) [inline tutorial/chapter_index.py]: Remove premium

Comment (xhlulu) [reply tutorial/chapter_index.py]: Actually, i think the description should be changed something along the line of "Beautifully styled technical components for engineering and physics", depending on what's the target audience for the component.

File: tutorial/daq_component_examples.py [hunk 2]
```diff
@@ -0,0 +1,1117 @@
...
+import dash_html_components as html
+from textwrap import dedent as s
+
+from tutorial import styles
+from tutorial import tools
+from tutorial.utils.convert_props_to_table_daq import generate_prop_table
+from tutorial.utils.component_block import ComponentBlock
+from tutorial.components import Syntax, Example
+
+examples = {
```
[the remark this answers is not in the document]
Comment (shammamah) [reply tutorial/daq_component_examples.py]: @xhlulu How exactly do you mean?

File: tutorial/daq.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    # Dash DAQ 
+
+    Dash is a web application framework that provides pure Python abstraction 
+    around HTML, CSS, and JavaScript.
+
+    Dash DAQ comprises a robust set of controls that make it simpler to 
+    integrate data acquisition and controls into your Dash applications. 
+
+    The source is on GitHub at [plotly/dash-daq](https://github.com/plotly/dash-daq). 
+
```
[the remark this answers is not in the document]
Comment (shammamah) [reply tutorial/daq.py]: @xhlulu I was going for something similar to what is already there for `dcc`: https://dash.plot.ly/dash-core-components/

I could include the little code snippet they have below, though! Let me know what you think.

**Comment by Marc-Andre-Rivet** [comment]
@cldougl @xhlulu @shammamah This merge seems to be the source cause of the broken tests for this repo. Please fix.

**Comment by cldougl** [comment]
@Marc-Andre-Rivet not 100% sure this was the cause -- issue was logged here https://github.com/plotly/dash-docs/issues/359 for further investigation

**Pull request merged**